pkg/util/oc/oc.go

```diff
@@ -82,6 +82,24 @@ func CreateNamespace(t test.TestHelper, ns string) {
 	shell.Executef(t, "oc new-project %s", ns)
 }
 
+func CreateNamespaces(t test.TestHelper, yaml string, data interface{}) {
+	t.T().Helper()
+	t.Logf("Creating namespaces")
+	ApplyTemplate(t, "default", yaml, data)
+}
+
+func DeleteNamespaces(t test.TestHelper, yaml string, data interface{}) {
+	t.T().Helper()
+	t.Logf("Deleting multiple namespaces")
+	DeleteFromTemplate(t, "default", yaml, data)
+}
```

Comment:
This doesn't look great. When I see a function called `CreateNamespaces` or `DeleteNamespaces`, I expect it to have a parameter where I specify the names of the namespaces to create/delete, not a YAML and data (I wouldn't know what the YAML is supposed to be; and I can pass in YAML that has nothing to do with namespaces).

If you change the function to `CreateTestNamespaces(t test.TestHelper, count int)`, it could stay here, but otherwise, I'd move it into the test file, as it's too specific to be a part of `oc`. Same for `DeleteNamespaces`.

Reply:
In data I pass the Count with the number of namespaces to create:
```go
data := map[string]interface{}{
			"Count":     50,
			"Namespace": meshNamespace,
		}
```
I'm going to move the template for multiple namespaces to inside the function to avoid this confusions

Reply:
To simplify, I move it the func to the test file because is too specific